### ISSUE TITLE
Input and output of the trained model

### ISSUE THREAD

**Issue by phongnhhn92** [kwotsin/transfer_learning_tutorial#17]
Hello guys, I followed your code to train my model and this is the result.
![screenshot from 2018-02-06 18-41-22](https://user-images.githubusercontent.com/11288381/35837699-f89e5d64-0b2a-11e8-9a01-a5f11874c83a.png)
I have one question though, what is the input and output node of the trained model ? I need those two information to convert it to another format in order to execute that trained model on mobile device.
Can anyone help me ?

**Comment by Karthik777** [comment]
+1
Trying to freeze the graph and use it with tensorflow-cpp.
Update: Short answer: 
input layer =  "batch" 
output layer = "InceptionResnetV2/Logits/Predictions"

**Comment by kwotsin** [comment]
As the input/output name may vary from model to model, I find the best way to check it is to simply use the following:
```
for op in graph.get_operations:
    print op.values()
```
in the context of the graph you defined locally. For an example, refer to this document: https://github.com/kwotsin/TensorFlow-Xception/blob/master/write_pb.py
which gives you an example of how to freeze it. Print the `op.values()` in the graph context as defined in the python file.

**Comment by kwotsin** [comment]
Closing due to inactivity.
